local_modules/execer/execer.cc

```diff
@@ -15,17 +15,56 @@
 #include <dirent.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <time.h>
 #include <node.h>
+#include <sstream>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string>
 #include <sys/socket.h>
+#include <sys/stat.h>
 #include <unistd.h>
 #include <v8.h>
 #include <vector>
 
 using namespace v8;
 
+bool write_response(int fd) {
+	// This is what Nginx does.
+	char buf[2048];
+	time_t now = time(0);
+	struct tm gm = *gmtime(&now);
```

Comment:
I would always avoid the non-r variants out of an abundance of caution, but you are right this is fine.

It sounds like OS X has gmtime_r: https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man3/gmtime_r.3.html